server/consensus/consensus.go

```diff
@@ -81,7 +81,8 @@ var (
 
 type IConsensus interface {
 	SyncGetImage() (*image.RGBA, error)
-	SyncGetLastUserModification(userId string) (time.Time, error)
+	SyncUpdatePixel(x, y, r, g, b, a int) error
+	SyncGetLastUserModification(userId string) (*time.Time, error)
```

Comment:
@jeffrey990219 i converted it back to a pointer because i didn't like returning a full `time.Time` object if the function resulted in an error.  using a pointer let's us return a value that the caller CANNOT use if there was an error, which should be safer.

Reply:
yea it's fine. I believe the backend is treating it as a pointer and dereference it if it's not nil